apps/interactor/interactor/addon.py

```diff
@@ -61,3 +63,40 @@ async def migrate(collector, extra=None, **kwargs):
     if extra is None:
         extra = collector.configuration["photons_app"].extra
     await database.migrate(collector.configuration["interactor"].database, extra)
+
+
+@an_action(label="Interactor")
+async def bind_address(collector, **kwargs):
```

Comment:
```suggestion
async def interactor_address(collector, **kwargs):
```

Reply:
Considering you vetoed the use of a task to inform the `command` script, the task itself is now redundant. So I'll just remove it completely.